github/migrations_issues_import_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestIssueImportService_Create(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	now := time.Now()
+	input := &IssueImportRequest{
+		IssueImport: IssueImport{
+			Assignee:  String("developer"),
+			Body:      "Dummy description",
+			CreatedAt: &now,
+			Labels:    []string{"l1", "l2"},
+			Milestone: Int(1),
+			Title:     "Dummy Issue",
+		},
+		Comments: []*Comment{&Comment{
+			CreatedAt: now,
+			Body:      "Comment body",
+		}},
+	}
+
+	mux.HandleFunc("/repos/o/r/import/issues", func(w http.ResponseWriter, r *http.Request) {
+		v := new(IssueImportRequest)
+		json.NewDecoder(r.Body).Decode(v)
+		testMethod(t, r, "POST")
+		testHeader(t, r, "Accept", mediaTypeIssueImportAPI)
+		if !cmp.Equal(v, input) {
```

Comment:
Tests with `reflect.DeepEqual` would fail. I did research about this and apparently the culprit of this failure would be `time` structs.

Reply:
Time structs are pervasive throughout this repo... you probably don't want them or need them in your unit tests... Please refer to the other examples in this repo to see how they are handled. Thanks.